src/transformers/generation/configuration_utils.py

```diff
@@ -472,6 +472,11 @@ def validate(self, is_init=False):
             raise ValueError(f"`early_stopping` must be a boolean or 'never', but is {self.early_stopping}.")
         if self.max_new_tokens is not None and self.max_new_tokens <= 0:
             raise ValueError(f"`max_new_tokens` must be greater than 0, but is {self.max_new_tokens}.")
+        if self.pad_token_id is not None and self.pad_token_id < 0:
+            warnings.warn(
+                f"`pad_token_id` must be positive but got {self.pad_token_id}. This may cause erros when batch generating. "
+                "Please set `pas_token_id` explicitly by `model.generation_config.pad_token_id=PAD_TOKEN_ID` to avoid errors in generation"
```

Comment:
```suggestion
                "Please set `pas_token_id` explicitly by `model.generation_config.pad_token_id=PAD_TOKEN_ID` to avoid errors in generation, and ensure your `input_ids` input does not have negative values."
```
Fixing the `generation_config` may be insufficient, if the input is already padded with incorrect values 🤗

Reply:
ah, that also might be the case. The one mentioned only had `-1` in model's config